test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java

```diff
@@ -168,6 +170,12 @@ private ExtensionState doJavaStart(ExtensionContext context, Class<? extends Qua
                 rootBuilder.add(appResourcesLocation);
             }
 
+            Path root = Paths.get("").normalize().toAbsolutePath();
+            // If gradle project running directly with IDE
+            if (System.getProperty(BootstrapConstants.SERIALIZED_APP_MODEL) == null) {
```

Comment:
Where is this property set? I see it in `exportModel` but has it been called by this point?

Reply:
It could already have been set by the gradle plugin in the before test

Reply:
We'll need a delegating impl of AppModelResolver. For now it's ok, I think.